packages/components/src/InputTime/InputTimeSafari.tsx

```diff
@@ -20,12 +20,12 @@ export function InputTimeSafari({
   ...params
 }: InputTimeProps) {
   const inputTime = React.createRef<HTMLInputElement>();
+  const debouncedHandleChange = debounce(handleChange, 1000);
+  const changeHandler = generateEventHandler(debouncedHandleChange);
+  const blurHandler = generateEventHandler(handleChange);
```

Comment:
Should we pass these in as the dependencies array into `useLayoutEffect `? I believe `eslint-plugin-react-hooks` would complain if they aren't included. Are we planning to include the plugin? https://www.npmjs.com/package/eslint-plugin-react-hooks